### DISCUSSION TITLE
Further functionality

### DISCUSSION THREAD

**Discussion by christopher-fradley** [sveinungf/spreadcheetah#4] [Q&A] [answered]
What new features are planned in the future?
And would it be poossible to integrate a cell merge funtionality?

Thank you very much for this package and the maintaing of it. Super simple and easy to use.

Cheers, Chris

**Comment by sveinungf** [comment, marked as answer]
Thanks for the feedback!
More commonly used features will typically be prioritized first. Requesting a feature is a good way of letting me know what people need. Merging cells is something I'll see if can be added for the next version.

**Reply by christopher-fradley** [reply]
Nice! :)
Thanx for the answer.
Where can I post my feature requests? Under Ideas maybe?

Thanx for the new borders feature 👍

**Reply by sveinungf** [reply]
Yes you can post feature requests under Ideas 👍
